docs/fargate.md

```diff
@@ -0,0 +1,438 @@
+    --file-system-id "${EFS_ID}" \
+    --subnet-id "${subnet}" \
+    --security-group "${SG_ID}" \
+    --region "${AWS_REGION}"
+done
+```
+
+#### Create `sumo-metrics-pvc.yaml` with PVC per access point
+
+After creation Access Points and ensuring that they have been authorized, the following Kubernetes storage objects have to be created:
+
+- `StorageClass` which informs AWS to bind EFS Access Points with Kubernetes Persitence Volume objects
+- `PersistentVolume` which is a Kubernetes representation of the EFS Access Point
+- `PersistenceVolumeClaim` which is a user request to use the storage
+
+The objects can be created using the following `bash` script:
+
+```bash
+## Create `sumo-metrics-pvc.yaml` with PVC per access point
+echo "---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: efs-sc
+provisioner: efs.csi.aws.com
+volumeBindingMode: Immediate" | tee sumo-metrics-pvc.yaml
+
+for (( counter=0; counter<$METRIC_PODS; counter++ )); do
+  FSAP_ID="$(
+    aws efs describe-access-points \
+      --region "${AWS_REGION}" | \
+      jq ".AccessPoints[] |
+        select(.RootDirectory.Path == \"/${NAMESPACE}/file-storage-${HELM_INSTALLATION_NAME}-sumologic-otelcol-metrics-${counter}\") |
+        .AccessPointId" \
+        --raw-output)"
+
+  echo "---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: file-storage-${HELM_INSTALLATION_NAME}-sumologic-otelcol-metrics-${counter}
+  labels:
+    app: ${HELM_INSTALLATION_NAME}-sumologic-otelcol-metrics
+spec:
+  capacity:
+    storage: 10Gi
+  volumeMode: Filesystem
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: efs-sc
+  claimRef:
+    namespace: ${NAMESPACE}
+    name: file-storage-${HELM_INSTALLATION_NAME}-sumologic-otelcol-metrics-${counter}
+  csi:
+    driver: efs.csi.aws.com
+    volumeHandle: ${EFS_ID}::${FSAP_ID}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: file-storage-${HELM_INSTALLATION_NAME}-sumologic-otelcol-metrics-${counter}
+  namespace: ${NAMESPACE}
+  labels:
+    app: ${HELM_INSTALLATION_NAME}-sumologic-otelcol-metrics
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: efs-sc
+  resources:
+    requests:
+      storage: 5Gi"
+done | tee -a sumo-metrics-pvc.yaml
+```
+
+#### Create namespace if it doesn't exist already
+
+```bash
+## Create namespace if it doesn't exist already
+kubectl create namespace "${NAMESPACE}"
+```
+
+#### Apply `sumo-metrics-pvc.yaml`
+
+```bash
+## Apply `sumo-metrics-pvc.yaml`
+kubectl apply -f sumo-metrics-pvc.yaml -n "${NAMESPACE}"
+```
+
+### Install or upgrade collection
+
+Disable Prometheus Node Exporter. As Amazon Fargate does not support Daemonsets, they won't be working correctly. There is no need to have
+Node Exporter on Amazon Fargate as Amazon takes care of Nodes management. To disable Prometheus Node Exporter, please add the following
+configuration to `user-values.yaml`:
+
+```yaml
```

Comment:
I think we also need this in our use-values.yaml:

```
sumologic:
  collector:
    logs:
      collector:
        otelcol:
          enabled: false
```

Reply:
yeah, but this specific section is about metrics signal

Reply:
If we don't want to merge unfinished document we can work on this branch, but lets keep it consistent, so refactor it to write about all signals the same time or split it into sections. Now it's splited, but maybe it would be better to slightly reorganize it

Reply:
any of the mentioned change should be subject of separate PR in my opinion